src/db/migrations/20190728221432-create-roles.js

```diff
@@ -0,0 +1,42 @@
+import Debug from 'debug';
+
+const debug = Debug('dev');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    try {
+      await queryInterface.createTable('Article', {
```

Comment:
The migration file name is `create-roles` while the table name is `Article` which should be `Articles` by the way. Are we missing something?